pages/docs/cloud.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Grouparoo Cloud Integration"
+date: "2021-11-08"
+pullQuote: "An easy-to-follow guide on integrating your Code Config repository with Grouparoo Cloud"
+---
+
+We're excited to share our Cloud offering! This guide should cover everything you need to get integrated
+using your own repository with Grouparoo Cloud.
+
+### Step 1 - Acquiring an API Key
+
+As of right now you need to make a [Trial Request](/trial) in order to get Grouparoo Cloud access. Our Administrators should get back to you within a day to help get you set up in our system and ready to integrate! After the setup process you should have an API Key and a Project ID which are required for integration.
```

Comment:
Perhaps update this to reflect that the API Key can be acquired from the Integrations page

also, maybe check with @andyjih on the "SLA" or whatever the right term is for getting back to them (referring to `Our Administrators should get back to you within a day`)

Reply:
Yeah, +1 to removing the timeline SLA. Just saying signing up for a Trial Request will get you access to the API Key is fine for now.

Reply:
```suggestion
To get started, you need to make a [Trial Request](/trial) to get Grouparoo Cloud access. Our Administrators will help get you set up in our system and ready to integrate! After the setup process you will have an API Key and a Project ID which are required for integration.
```